xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.22.xml

```diff
@@ -108,7 +109,14 @@
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
```

Comment:
I've added it in myself, but remember to but `@key` on `<name>` so that we can pull the different names in an index.

Reply:
Also I think we normally use gentilicium instead of nomen for that name? We can confirm and the go back to update.